src/plugins/exec/sim-host-handlers.js

```diff
@@ -87,3 +96,19 @@ function handleSavedSim(success, fail, service, action) {
     }
     return false;
 }
+
+function sendExecUnhandledTelemetry(service, action, hasPersisted, isSuccess, hasResult, saveResult) {
```

Comment:
It would be really helpful if we add some jsdoc to the functions, at least the parameters types, and which of them are optional if any.